crypto/ec_extra/ec_asn1.c

```diff
@@ -614,3 +614,44 @@ BIGNUM *EC_POINT_point2bn(const EC_GROUP *group, const EC_POINT *point,
 
   return ret;
 }
+
+EC_POINT *EC_POINT_bn2point(const EC_GROUP *group,
+                            const BIGNUM *bn, EC_POINT *point, BN_CTX *ctx) {
+  EC_POINT *ret = NULL;
+
+  // Allocate buffer and length.
+  size_t buf_len = BN_num_bytes(bn);
+  if (buf_len == 0) {
```

Comment:
why?? isn't this an error or something?

Reply:
Good question, I found the issue/commit on why this was done, but forgot to provide the link: https://github.com/openssl/openssl/issues/10258

I added it as a comment in the code.

Reply:
can we add a test for this case?

Reply:
Added